analyzers/src/SonarAnalyzer.CSharp/Extensions/InvocationExpressionSyntaxExtensions.cs

```diff
@@ -41,5 +41,17 @@ internal static class InvocationExpressionSyntaxExtensions
 
         internal static bool IsEqualTo(this InvocationExpressionSyntax first, InvocationExpressionSyntax second, SemanticModel model) =>
             model.GetSymbolInfo(first).Symbol.Equals(model.GetSymbolInfo(second).Symbol);
+
+        internal static bool TryGetOperands(this InvocationExpressionSyntax invocation, out SyntaxNode leftOperand, out SyntaxNode rightOperand)
```

Comment:
I think you can just call the out variables left and right, since the method is called TryGetOperands